examples/roam/src/pages/Blocks.js

```diff
@@ -0,0 +1,41 @@
+import { useQuery, useTransact } from 'homebase-react'
+import { nanoid } from 'nanoid'
+import React from 'react'
+import Block from '../components/Block'
+
+const Blocks = () => {
+  const [transact] = useTransact()
+  const [blocks] = useQuery({
+    $find: 'block',
+    $where: { block: { isNode: true } },
+  })
+
+  React.useEffect(() => {
+    setTimeout(() => {
```

Comment:
why are we using setTimeout?

Reply:
It's a hack to add a page for the current date if none exists. I'll leave a comment to that effect and if that's still too hacky we can refactor. Let me know what you think.